write/writer.go

```diff
@@ -384,7 +384,14 @@ func Element(e *dicomio.Encoder, elem *element.Element, opts ...Option) {
 			}
 			sube.WriteString(s)
 			if len(s)%2 == 1 {
-				sube.WriteByte(0)
+				switch vr {
+				// Values with VRs constructed of character strings, except in the case of the VR UI, shall be padded with SPACE characters
+				// per http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_6.2.html
```

Comment:
if you could update this link to the one from the current standard that'd be great! http://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_6.2

Reply:
Sure thing, done!